packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -386,6 +386,44 @@ class ShortcutManager extends ChangeNotifier with Diagnosticable {
 /// when invoking an [Action] via a keyboard key combination that maps to an
 /// [Intent].
 ///
+/// {@tool dartpad --template=stateful_widget_material}
+/// ```dart imports
+/// import 'package:flutter/services.dart';
+/// ```
+///
+/// ```dart preamble
+/// class Increment extends Intent {}
+///
+/// class Decrement extends Intent {}
+/// ```
+///
+/// ```dart
+/// int count = 0;
+///
+/// Widget build(BuildContext context) {
+///   return Shortcuts(
+///     shortcuts: <LogicalKeySet, Intent> {
+///       LogicalKeySet(LogicalKeyboardKey.alt, LogicalKeyboardKey.keyI): Increment(),
+///       LogicalKeySet(LogicalKeyboardKey.alt, LogicalKeyboardKey.keyD): Decrement(),
```

Comment:
Maybe use different key sets, because alt + d is already a shortcut for debugging on the web, so there's a conflict there. Maybe the + and - signs on the keyboard instead?
```suggestion
        LogicalKeySet(LogicalKeyboardKey.shift, LogicalKeyboardKey.equal): Increment(),
        LogicalKeySet(LogicalKeyboardKey.minus): Decrement(),
        LogicalKeySet(LogicalKeyboardKey.numpadAdd): Increment(),
        LogicalKeySet(LogicalKeyboardKey.numpadSubtract): Decrement(),
```